understanding/21/orientation.html

```diff
@@ -31,11 +31,11 @@ <h3>Benefits</h3>
 			<h2>Examples</h2>
 			<ul>
 				<li><strong>Example 1: Online video site</strong><br />
-				Videos are not locked to a specific orientation.  A user can start a video with their device in a portrait orientation, resulting in a smaller video being displayed and then rotate the device to landscape and have the video expand to fill the entire screen.  All controls and functionality are available in either orientation.</li>
+				Videos are not locked to a specific orientation.  A user can start a video with their device in a portrait orientation or in landscape.</li>
 				<li><strong>Example 2: Messaging website</strong><br />
 				A messaging website can display messages in both portrait and landscape orientations without a loss of content or functionality. </li>
 				<li><strong>Example 3: eReader web app</strong><br />
-				An eReader web app can display the contents of a book in both portrait and landscape mode without a loss of content or functionality</li>
+				An eReader web app can display the contents of a book in both portrait and landscape.</li>
```

Comment:
needs a word, either "mode" or "orientation".

Also, I think you missed the one above, see line 36.